content/sdk/10.js/00.ethers/20.v6/00.getting-started.md

```diff
@@ -1,69 +1,134 @@
 ---
 title: "Getting Started"
 description: "Explore zkSync-ethers SDK version 6"
-tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "javascript", "v6"]
+tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "javascript", v6"]
 ---
 
-While most of the existing SDKs should work out of the box, deploying smart contracts or using unique zkSync features,
-like account abstraction, requires providing additional fields to those that Ethereum transactions have by default.
+While most existing SDKs work out of the box, deploying smart contracts or using unique zkSync features like account
+abstraction requires additional fields beyond the default Ethereum transaction fields. Understanding the types of objects
+and their responsibilities at a high level is useful:
 
-To begin, it is useful to have a basic understanding of the types of objects available and what they are
-responsible for, at a high level:
+- **Provider**: Connects to the zkSync Era blockchain, allowing querying of account, block, or transaction details, event
+logs, or read-only code evaluation using `call`. It also facilitates writing to the blockchain by sending transactions.
+- **Wallet**: Manages all operations interacting with an account, which typically has a private key for signing various
+payloads. It simplifies the use of common features.
 
-- `Provider` provides connection to the zkSync Era blockchain, which allows querying the blockchain state,
-  such as account, block or transaction details, querying event logs or evaluating read-only code using call.
-  Additionally, the client facilitates writing to the blockchain by sending transactions.
-- `Wallet` wraps all operations that interact with an account. An account generally has a private key, which
-  can be used to sign a variety of types of payloads. It provides easy usage of the most common features.
+::callout{icon="i-heroicons-light-bulb"}
+This example uses zkSync JavaScript SDK to interact with the zkSync Era blockchain.
+::
 
-## Examples
+- Create the scripts folder inside your project directory:
 
-The following examples could be easily run by writing the code snippets in a file and executing them as follows:
+```bash
+touch scripts
+```
 
-```shell
-npx ts-node src/zksync.ts
+- Create a file inside the scripts folder:
+
+```bash
+touch scripts/zksync.ts
 ```
 
-Connect to the zkSync Era network:
+### Importing Modules
+
+```javascript
+import { Provider, types } from "zksync-ethers";
+```
 
-```ts
-import { Provider, utils, types } from "zksync-ethers";
-import { ethers } from "ethers";
+- **Provider**: A module to interact with the zkSync network.
+- **types**: A module containing type definitions, including network identifiers.
 
-const provider = Provider.getDefaultProvider(types.Network.Sepolia);
-const ethProvider = ethers.getDefaultProvider("sepolia");
+### Main Function
+
+```javascript
+async function main() {
```

Comment:
The main function doesn't need to be described, it's just one way of writing and organizing this code. The closing bracket is missing as well. If this is to indicate the rest of the code is a part of the main function, we shouldn't break up the code like this. It makes it extremely hard to know where the rest of the code goes as someone is reading this guide.

Consider putting the code back together and using comments for your descriptions. For example like the Ethers.js doc's Getting Started code samples.

Reply:
Yes, those blocks of code are broken down, but they are complete and runnable within their own context. The main function cannot run in the state that it is presented in this block.

Reply:
Okay, I understand..

I'll do that now